internal/provisioner/controller/gatewayclass_test.go

```diff
@@ -266,6 +268,16 @@ func TestGatewayClassReconcile(t *testing.T) {
 						NetworkPublishing: &contourv1alpha1.NetworkPublishing{
 							Type: "invalid-networkpublishing-type",
 						},
+						ExtraVolumeMounts: []corev1.VolumeMount{
+							{
+								Name: "volume-a",
+							},
+						},
+						ExtraVolumes: []corev1.Volume{
+							{
+								Name: "volume-b",
+							},
+						},
```

Comment:
This should go in a separate test case - the case you edited is testing for an invalid NetworkPublishing type. Please create a separate one specifically for ExtraVolumes/ExtraVolumeMounts.